third_party/docfx-doclet-143274/pom.xml

```diff
@@ -17,6 +17,7 @@
     <apache.commons-lang.version>3.12.0</apache.commons-lang.version>
     <apache.commons-collections.version>4.4</apache.commons-collections.version>
     <apache.commons-io.version>2.11.0</apache.commons-io.version>
+    <apache.commons-text.version>1.10.0</apache.commons-text.version>
```

Comment:
This version is safe. But commons text is often marked as CVEs.

https://security.netapp.com/advisory/ntap-20221020-0004/